vendor/bat-native-ads/src/bat/ads/internal/processors/contextual/text_embedding/text_embedding_event_unittest_util.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_PROCESSORS_CONTEXTUAL_TEXT_EMBEDDING_TEXT_EMBEDDING_EVENT_UNITTEST_UTIL_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_PROCESSORS_CONTEXTUAL_TEXT_EMBEDDING_TEXT_EMBEDDING_EVENT_UNITTEST_UTIL_H_
+
+#include "bat/ads/internal/processors/contextual/text_embedding/text_embedding_event_info.h"
```

Comment:
not needed as you already forward declare

Reply:
Assuming you mean to remove it from the .cc file. Removed it there